src/wikibots/flickr.py

```
@@ -0,0 +1,59 @@
+import os
+import sys
+from typing import Any
+
+from pywikibot import Claim
+from pywikibot.pagegenerators import SearchPageGenerator
+
+try:
+    sys.path.append(os.path.dirname(os.path.abspath(__file__)) + '/lib')
+    from lib.bot import BaseBot
+    from lib.wikidata import WikidataEntity, WikidataProperty
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)